### PR TITLE
setup - db connect, models initialized

### PR DESCRIPTION
**Pull request by thapasya-m**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-10-tripen/wiki/Commit-message-guidelines)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (compulsory for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [x] Feature
- [ ] Tests changed / added / updated
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Dependency upgrade
- [ ] Other... Please describe:

## Issue Number: 
#

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

File: server/src/configs/database.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import mongoose from 'mongoose';
+import config from '../configs/env';
+
+const uri = config.databaseURI;
+
+mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
```
Comment (SarthakC) [inline server/src/configs/database.js]: Have you checked this?
https://mongoosejs.com/docs/deprecations.html

Comment (thapasya-m) [reply server/src/configs/database.js]: yes, after that deprecation warning i added useUnifiedtopology: true

**Comment by thapasya-m** [comment]
> Just curious. Why haven't we used ES5 import syntax?

exactly which file are you referring? :/

File: server/src/configs/database.js (hunk 1 above)
Comment (karansapolia) [reply server/src/configs/database.js]: are createIndex errors not being thrown? Usually, I need `useCreateIndex: true` also in mongoose.connect

File: server/src/controllers/timeline.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const getAllTimeline = (req, res) => {
+    res.status(200).json({ 'status': 'fetch all timelines' });
```
Comment (karansapolia) [inline server/src/controllers/timeline.js]: What about the controllers logic?

File: server/src/configs/database.js (hunk 1 above)
Comment (thapasya-m) [reply server/src/configs/database.js]: not yet thrown

File: server/src/controllers/timeline.js (hunk 2 above)
Comment (thapasya-m) [reply server/src/controllers/timeline.js]: i will be adding the impltn in new PR

File: server/src/configs/dbHelper.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+let mongod;
+async function connect() {
+    mongod = new MongoMemoryServer();
```
Comment (nimish-gupta) [inline server/src/configs/dbHelper.js]: This memory server is good for testing but not for real time applications, you have to use a mongodb which is hosted at some place.

Comment (thapasya-m) [reply server/src/configs/dbHelper.js]: yes, this is for testing alone. db url is stored in .env file. i will rename the file.

**Pull request merged**